common/src/main/java/com/wynntils/gui/screens/maps/MainMapScreen.java

```diff
@@ -220,12 +226,59 @@ private void renderMap(PoseStack poseStack, int mouseX, int mouseY) {
         hovered = null;
 
         renderPois(poseStack, textureBoundingBox, mouseX, mouseY);
+        renderPlayerIcons(poseStack, textureBoundingBox, mouseX, mouseY);
+
         // Cursor
         renderCursor(poseStack);
 
         RenderSystem.disableScissor();
     }
 
+    private void renderPlayerIcons(PoseStack poseStack, BoundingBox textureBoundingBox, int mouseX, int mouseY) {
+        for (HadesUser user : HadesUserModel.getHadesUserMap().values()) {
```

Comment:
Can player icons be made into a POI?

Reply:
ATM Pois have a fixed location. As these are meant to be updated frequently, I chose to do this, to save a lot of allocations. I think this is candidate for a POI refactor (as we would need to update POI locations for player-defined POIs aswell)

Reply:
Maybe POIs could use a supplier or something. An arguement for this is that we should be able to place a compass on player icons and making them POIs would give that easily.

Reply:
But you agree that this is fine as a workaround for now, so we do not have to change POI infrastructure in an already big PR?

Reply:
Yeah, best for a separate pr that does suppliers + caching